symphony-bdk-core/src/test/resources/config/config_client_global.yaml

```diff
@@ -0,0 +1,21 @@
+host: devx1.symphony.com
+context: context
+port: 8443
+
+pod:
+  port: 8443
+
+agent:
+  host: devx1.symphony.com
+  port: 443
+
+keyManager:
+  host: devx1.symphony.com
```

Comment:
You add a test case where the `context` is different

Reply:
Added 👍